solutions/usaco-810.mdx

```diff
@@ -0,0 +1,148 @@
+---
+id: usaco-810
+source: USACO Silver 2018 Silver
+title: Rest Stops
+author: Vivian Han
+---
+
+## Explanation
+
+Suppose that early in the hike there is a rest stop with tastiness $c$, but later there is a rest stop with tastiness $C$, where $C>c$. Then it is never optimal for Bessie to spend any time at the first rest stop, with tastiness $c$. If she did, she could spend the same amount of time at the later rest stop instead and earn more tastiness units. Doing this, she would still never be behind Farmer John. So the only rest stops which Bessie might stop at are the ones which have more tastiness than any subsequent rest stops.
```

Comment:
maybe different variable names for `c` and `C`?

Reply:
this